service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java

```diff
@@ -43,793 +56,941 @@
 import org.springframework.beans.factory.annotation.Autowired;
 
 public class CreateAndDeleteAzureControlledResourceFlightTest extends BaseAzureTest {
```

Comment:
@sergiygetlin FYI that this repo uses indenting of 2 spaces (`google-java-format`). I am re-indenting this file in my open PR, but I wanted to let you know for future commits. Indenting changes like this cause merge conflicts, which can be painful, and they make the commit history more difficult to follow.

Thanks!